### PR TITLE
Fix minor nits in the nuxeo tests which were causing them to fail

### PR DESCRIPTION
**Pull request by tianon**
cc @dmetzler

### CODE REVIEW

File: test/tests/nuxeo-basics/run.sh [hunk 1]
```diff
@@ -15,23 +15,33 @@ trap "docker rm -vf $cid > /dev/null" EXIT
...
 
-. "./../../retry.sh" --tries "$NUXEO_TEST_TRIES" \
-	--sleep "$NUXEO_TEST_SLEEP" \
-	"get default-domain/workspaces"
+python3() {
+	docker run --rm -i \
+		--entrypoint python3 \
+		"$image" \
+		"$@"
+}
```
Comment (tianon) [inline test/tests/nuxeo-basics/run.sh]: We don't have `python` in the test environment, so we get to be a little creative. :smile:

**Comment by tianon** [comment]
Heh, looks like we need a sort, also:
```
	'nuxeo-conf' [5/6]...failed; unexpected output:
--- /usr/src/pr/test/tests/nuxeo-conf/expected-std-out.txt
+++ -
@@ -1,2 +1,2 @@
-org.nuxeo.automation.trace=true
 org.nuxeo.dev=true
+org.nuxeo.automation.trace=true
```

**Comment by tianon** [comment]
Whooo!

Build test of #1302; 548adc7 (`nuxeo`):

```console
$ bashbrew build "nuxeo"
Cloning nuxeo (git://github.com/nuxeo/docker-nuxeo) ...
Processing nuxeo:latest ...
Processing nuxeo:LTS-2015 ...
Processing nuxeo:LTS ...
Processing nuxeo:7.10 ...
Processing nuxeo:7 ...
Processing nuxeo:7.4 ...
Processing nuxeo:LTS-2014 ...
Processing nuxeo:6 ...
Processing nuxeo:6.0 ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing nuxeo:latest
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6]....passed
testing nuxeo:7.4
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6]....passed
testing nuxeo:LTS-2014
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6]....passed
```

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**